feature-staking-impl/src/main/java/io/novafoundation/nova/feature_staking_impl/domain/nominationPools/redeem/validations/ValidationFailureUi.kt

```diff
@@ -2,17 +2,19 @@ package io.novafoundation.nova.feature_staking_impl.domain.nominationPools.redee
 
 import io.novafoundation.nova.common.base.TitleAndMessage
 import io.novafoundation.nova.common.resources.ResourceManager
-import io.novafoundation.nova.feature_staking_impl.R
 import io.novafoundation.nova.feature_staking_impl.domain.nominationPools.redeem.validations.NominationPoolsRedeemValidationFailure.NotEnoughBalanceToPayFees
 import io.novafoundation.nova.feature_wallet_api.domain.validation.handleNotEnoughFeeError
+import io.novafoundation.nova.feature_wallet_api.presentation.validation.handleInsufficientBalanceCommission
 
 fun nominationPoolsRedeemValidationFailure(
     failure: NominationPoolsRedeemValidationFailure,
     resourceManager: ResourceManager
 ): TitleAndMessage {
     return when (failure) {
         is NotEnoughBalanceToPayFees -> handleNotEnoughFeeError(failure, resourceManager)
-        is NominationPoolsRedeemValidationFailure.ToStayAboveED -> resourceManager.getString(R.string.common_not_enough_funds_title) to
-            resourceManager.getString(R.string.wallet_send_insufficient_balance_commission, failure.asset.symbol)
+        is NominationPoolsRedeemValidationFailure.ToStayAboveED -> handleInsufficientBalanceCommission(
+            failure.asset,
```

Comment:
I suggest to pass `failure` directly to `handleInsufficientBalanceCommission`. For that, we can create an interface `InsufficientTotalToStayAboveEDError` which contains `all fields necessary for displaying an error. This will be useful in case we want to change display logic and add/remove extra information

Reply:
I think it's not really easy to provide this fix since we use different failure classes